chartfx-chart/src/main/resources/de/gsi/chart/solid-pick.css

```diff
@@ -119,269 +119,202 @@ is being used to size a border should also be in pixels.
-    /* 70% opacity */
-    CHART_COLOR_1_TRANS_70: #f3622db3;
-    CHART_COLOR_2_TRANS_70: #fba71bb3;
-    CHART_COLOR_3_TRANS_70: #57b757b3;
-    CHART_COLOR_4_TRANS_70: #41a9c9b3;
-    CHART_COLOR_5_TRANS_70: #4258c9b3;
-    CHART_COLOR_6_TRANS_70: #9a42c8b3;
-    CHART_COLOR_7_TRANS_70: #c84164b3;
-    CHART_COLOR_8_TRANS_70: #888888b3;
-
-    /***************************************************************************
+  chart_color_1_trans_20: #f3622d33;
+  chart_color_2_trans_20: #fba71b33;
+  chart_color_3_trans_20: #57b75733;
+  chart_color_4_trans_20: #41a9c933;
+  chart_color_5_trans_20: #4258c933;
+  chart_color_6_trans_20: #9a42c833;
+  chart_color_7_trans_20: #c8416433;
+  chart_color_8_trans_20: #88888833;
+
+  /* 70% opacity */
+  chart_color_1_trans_70: #f3622db3;
+  chart_color_2_trans_70: #fba71bb3;
+  chart_color_3_trans_70: #57b757b3;
+  chart_color_4_trans_70: #41a9c9b3;
+  chart_color_5_trans_70: #4258c9b3;
+  chart_color_6_trans_70: #9a42c8b3;
+  chart_color_7_trans_70: #c84164b3;
+  chart_color_8_trans_70: #888888b3;
+
+  /***************************************************************************
      *                                                                         *
      * Colors that are derived from the main color palette.                    *
      *                                                                         *
      **************************************************************************/
 
-    /* A little lighter than -fx-base and used as the -fx-color for the
+  /* A little lighter than -fx-base and used as the -fx-color for the
      * "hovered" pseudoclass state.
      */
-    -fx-hover-base: ladder(
-        -fx-base,
-        derive(-fx-base,20%) 20%,
-        derive(-fx-base,30%) 35%,
-        derive(-fx-base,40%) 50%
-     );
-
-    /* A little darker than -fx-base and used as the -fx-color for the
+  -fx-hover-base: ladder(-fx-base, derive(-fx-base, 20%) 20%, derive(-fx-base, 30%) 35%, derive(-fx-base, 40%) 50%);
+
+  /* A little darker than -fx-base and used as the -fx-color for the
      * "armed" pseudoclass state.
      *
      * TODO: should this be renamed to -fx-armed-base?
      */
-    -fx-pressed-base: derive(-fx-base,-6%);
+  -fx-pressed-base: derive(-fx-base, -6%);
 
-    /* The color to use for -fx-text-fill when text is to be painted on top of
+  /* The color to use for -fx-text-fill when text is to be painted on top of
      * a background filled with the -fx-background color.
      */
-    -fx-text-background-color: ladder(
-        -fx-background,
-        -fx-light-text-color 45%,
-        -fx-dark-text-color  46%,
-        -fx-dark-text-color  59%,
-        -fx-mid-text-color   60%
-    );
-
-    /* A little darker than -fx-color and used to draw boxes around objects such
+  -fx-text-background-color: ladder(-fx-background, -fx-light-text-color 45%, -fx-dark-text-color 46%, -fx-dark-text-color 59%, -fx-mid-text-color
+    60%);
```

Comment:
Codacy found an issue: [Expected newline before ")" in a multi-line function (function-parentheses-newline-inside)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780860)